fhir-parent/pom.xml

```diff
@@ -47,7 +47,7 @@
         <fhir-search.index>MINIMAL_JSON</fhir-search.index>
         <fhir-persistence-jdbc.index>MINIMAL_JSON</fhir-persistence-jdbc.index>
         <fhir-server-test.index>MINIMAL_JSON</fhir-server-test.index>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```

Comment:
<source>8</source> update this in maven-javadoc-plugin

Reply:
Updated.